cms/templates/cms/home_page.html

```diff
@@ -160,5 +177,90 @@
   </div>
 
 </div>
+
+<!-- Modal subscribe alerts -->
+<div id="subscribeAlert" class="modal fade" role="dialog">
```

Comment:
just wondering: why is the `modal-about` modal ([here](https://github.com/mitodl/micromasters/pull/306/files#diff-2896e8df8c519be1744723a214c9d191R151)) nested within 2 divs, but this modal is not nested? i think it would be a good idea to define our modals at the same level and in close proximity. can we move `modal-about` to this level as a sibling of `modal-dialog`?